tests/page-objects/default/enketo/pregnancy.wdio.page.js

```diff
@@ -0,0 +1,210 @@
+const moment = require('moment');
+const genericForm = require('../../../page-objects/default/enketo/generic-form.wdio.page');
+const contactPage = require('../../../page-objects/default/contacts/contacts.wdio.page');
+
+const GESTATION_AGE = {lmp: 'method_lmp', lmpApprox: 'method_approx', edd: 'method_edd', none: 'none'};
```

Comment:
Adding some terminology to a global comment in this file would be helpful.
/**
* Terminology:
* lmp: Last menstrual cycle date is known (true or false)
* lmpApprox: Approximate date of last cycle
* edd: Estimate delivery date
*/

Reply:
Great! Thanks!